# contains an occupancy octree and a fringe octree

Header header
int8[] occupancy_data
int8[] fringe_data
